ar4_isaac/scripts/run_sim.py

```diff
@@ -44,23 +45,6 @@
 # this is required because the simulation app must be created first
 
 
-def create_simulation_context(
```

Comment:
Does this change works for you? When I try without it, Isaac crashes.

Reply:
Random comment: in my project, after enabing the ros2_bridge extension the script equivalent to this needs to do a couple of update() cycles on the app, or Isaac will crash randomly upon start. In Isaac 4.2.0 there seems to be a race condition when starting the ros extension.

Reply:
This seems to be working well! The only difference that I notice is that the simulation starts paused, but I don't think it's an issue.